engine/apps/public_api/views/escalation_chains.py

```diff
@@ -22,7 +21,7 @@ class EscalationChainView(RateLimitHeadersMixin, UpdateSerializerMixin, ModelVie
 
     model = EscalationChain
     serializer_class = EscalationChainSerializer
-    update_serializer_class = EscalationChainUpdateSerializer
+    update_serializer_class = EscalationChainSerializer
```

Comment:
You can just remove UpdateSerializerMixin and update_serializer_class from EscalationChainView and View will always serializer_class

Reply:
```suggestion
```